examples_tests/13.BRDF_Explorer/CBRDFBuiltinIncludeLoader.h

```diff
@@ -48,6 +48,14 @@ float GGXTrowbridgeReitz(in float a2, in float NdotH)
     return a2 / (3.14159265359 * denom*denom);
 }
 
+float GGXBurleyAnisotropic(float at, float ab, float TdotH, float BdotH, float NdotH) {
```

Comment:
This is correct

Reply:
Although I would pass `a2` and `anisotropy` (`uAnisotropy`) to this function.

This way you could optimize the computation like this
```cpp
float antiAniso = 1.0-anisotropy;
float atab = a2*antiAniso; //old a2
float anisoTdotH = antiAniso*TdotH;
float anisoNdotH = antiAniso*NdotH;
float w2 = antiAniso/(BdotH*BdotH+anisoTdotH*anisoTdotH+anisoNdotH*anisoNdotH*a2);
return w2*w2*atab / PI;
```

Reply:
@Crisspl wanna try ?

Reply:
@Crisspl ?

Reply:
left TODO
i think your code is missing `a2` factor and then it's ok

Reply:
but i'll check again tomorrow to be sure...

Reply:
doesn't `float atab = a2*antiAniso; //old a2` do the job? my `atab` is your `a2`

Reply:
no, i meant something different.
But seems like it's ok, it doesn't need anything